### ISSUE TITLE
Large Aggregate of Issues/Feedback From Reddit/GitHub/Forum

### ISSUE THREAD

**Issue by jaredeg** [ValveSoftware/Dota2-Gameplay#13382]
### Description

Putting together this post to aggregate many of the issues in the game that have been highlighted over time and post TI. Please comment below with details of anything I might have missed, or any suggested improvements.

&#x200B;

1. Matchmaking
   1. **Immortal Matchmaking** \- Immortal Matchmaking needs a version 2. It currently ignores all matchmaking options. This include - avoid a player functionality, Strict Solo Matchmaking, and Party Que. **Potential Improvement** \- An iteration of this would be to  only use Immortal Draft at the highest MMR games, or enforce Drafting Parties and those parties would expend party ques.
   2. **Smurfing/Account Buying** \- Accounts can now delete their overall matchmaking stats from their profiles. This means users have to look-up users on dotabuff/stratz, Steam Profile stats, and even checking Guilds, to really analyze if they are smurfing or not. If they are reported, nothing happens. This is especially relevant in party que as well as non-peak times (e.g. 9-5pm M-Fri on USE).  Please change the name of Smurfing to Smurfing/Account/Buying
      1. **Potential Improvement - 1.**  Let Smurf account reports be reviewed by overwatch. Give a panel of details on the account (and their potential similar accounts) for the reviewer. This can include, common heros played, mutual friends, servers played, # of games played, time of day played, etc. If the account gets flagged, it should have a badge on the account indicating its a potential smurf account.
      2. **Potential Improvement - 2.** Keep all account details in the account, included in these details should be (if a person is a high % of being a smurf) # of games  where they were reported for smurfing.
      3. **Potential Improvement - 3.** Communicate to reporters more regularly smurf indicators or false flags. If someone is reporting people that are clearly not smurfs, warn them, before they have a game limit on reports.
2. Behavior Score
   1. **Infinite Reports  -** Right now the "punishment" for excessively using the report system is not clear. We should have another score that indicates report accuracy. Ultimately if we are going to give scores to behavior & comms we should give scores to the enforcers as well.
      1. **Potential Improvement - 1.** The lower the report accuracy the more limits on the degree of reporting a user has. Maybe by default a report has a "high" impact on the reported user, and people with low accuracy can only have "low" impact.
      2. **Potential Improvement - 2.**  Maybe one of the more severe punishments could be that you can only report for a certain area AFTER someone (with a higher score) reports that user first.
   2. **Global Mutes Don't Follow New System** \- Details laid out [in my own post](https://www.reddit.com/r/DotA2/comments/17m3jta/why_was_the_behaviorcomm_score_update_abandoned/). But the TL;DR is that Communication Scores are not truly separated from Behavior Score. Comms can NEVER be higher than behavior score. Additionally regardless of your communication score you can be globally muted because you get enough reports and it has nothing to do with how much you chat or what you say, despite what the patch said. When you are globally muted you cant even /roll. If someone is muted and "griefing" their reports should all be reviewed by overwatch. This will heavily reduce peoples claims of being in the "trench"
      1. **Potential Improvement - 1.** Do not let enemies report enemies for comm abuse, just let them mute enemies.
      2. **Potential Improvement - 2.** If people report people for comms abuse that were previously muted, they should temporarily lose their ability to report for comms.
      3. **Potential Improvement - 3.** Make all comm bans based off of the behavior score and # of games played, not time based.
3. Bugs (I am sure there are other bugs, these are just the ones I have experienced and seen similar highlights)
   1. **Random Crashes are quite Frequent** \- MatchID  (7421615721) - had a crash the instant I used Magnus's first spell at time pointer around 15 seconds before bounties spawned. I have noticed these type of crashes happening more frequently, something that used to be rare before the large map update
   2. [**Users have to reverify game files**](https://github.com/ValveSoftware/Dota2-Gameplay/issues/8584) \- Since larger map update many of us are getting this issue, seemingly randomly.  I have a top of the line system and still get these issues. Previously this re-verify thing would happen after any crash. Now it happens more seldomly but typically after some sort of game-update. I have already completely reinstalled Dota 2
   3. **General Framerate Issues** \- I have a 5900X, 4090 and the latest drivers, and still run into random frame-rate issues. Kunka Aghs, MK Immortal Ult, are the most common ones. I think a great test Valve could run would be the Wraith King level in Aghs Lab... or the lag that was caused by Magnus skewering heroes off the Aghs Lab map. The dropped frames in these situations seems quite similar.
4. Gameplay
   1. **Tiny Airlines works Again -** Noticed this in a game where someone decided to dual mid with me in Turbo. They picked Tiny and followed me around. I reported them and nothing happened. I disabled help, but they could still throw me. This was previously patched that if someone disabled help the throw skill would go on CD.
   2. **Bottle Can Be Destroyed by Ally** \- Teammates can drop your bottle on the ground and destroy it.
   3. **Balance Patches -** Big Tournaments (Like TI) should be the culmination of a game patch. Things that are exposed to be OP should be "hot patched" out so the gameplay continues to be fun. We know this is possible because it is often done in the "fun" game modes like diretide/aghs lab. For example, CK had a high winrate for some time, and then in the most recent minor patch he got buffed, his winrate shot-up from \~54% to 58% This should be seen as a regression and should have been quickly mitigated.
      1. **Potential Improvement 1**. Keep the comms going, when the team posted the TI post/update there should have been an inclusion on balance patch coming after compendium closes (or something)
      2. **Potential Improvement 2** \- Minor patches should be used more regularly to balance the game, these should be pre-planned for after big tournaments.
   4. **Secondary Gamemode -** There is a strong sentiment across all channels that the most fun time to play dota is when there is a "chill" game mode outside of the main game. Aghs Lab is an example, but perhaps we can borrow an idea from Fortnite. They regularly rotate a few the most popular "Arcade" games into the main matchmaking. Perhaps we could consider doing something similar with a submission process. This also improves downtime between gameplay patches where people feel balance is poor.

&#x200B;

**Comment by envyvenvy** [comment]
fix the behavior that will be enough

**Comment by BledNavalny** [comment]
> fix the behavior that will be enough

Yeah, just revert august changes, all worked perfectly for 12 years( i dont like 2021 overwatch but still)
Or they doing this on purpose to not invent anything and revert old system after complaints
